persistent/Database/Persist/Types/Base.hs

```diff
@@ -242,8 +241,6 @@ data PersistException
     deriving (Show, Typeable)
 
 instance Exception PersistException
-instance Error PersistException where
```

Comment:
This is a breaking change. I'm not opposed to it in the future, but it shouldn't be undertaken immediately.